snippets/article-card.liquid

```diff
@@ -23,17 +23,17 @@
       assign ratio = media_aspect_ratio
     endif
   -%}
-  <div class="card-wrapper underline-links-hover">
+  <div class="article-card-wrapper card-wrapper underline-links-hover">
```

Comment:
I don't have very strong feelings towards doing this but I was wondering what your thoughts were on adding `card-wrapper--article` instead of `article-card-wrapper`?

```suggestion
  <div class="card-wrapper card-wrapper--article underline-links-hover">
```

We'd of course need to update the `base.css` selector from `.article-card-wrapper .card` to `.card-wrapper--article .card`

Reply:
Thanks Ken. Yeah, my thoughts were more or less the same as yours -- no strong feelings and you're probably right in thinking that it'd be too much at this point.

Reply:
That makes sense 🤔 I can keep it this way for now and we can change it in the future if needed.